### DISCUSSION TITLE
New CFEngine Webinar Series

### DISCUSSION THREAD

**Discussion by cody-valle** [cfengine/core#4658] [General]
Hi Everyone - If you haven't heard already, the CFEngine team is starting a new monthly webinar series titled, The Agent Is In. This series is intended for both CFEngine Community & Enterprise users who are looking to take their knowledge to the next level. Each month we will share tips & tricks, best practices, new/upcoming features, Q&A, and more. 

Our first webainr will be on Thursday, May 27th at 9AM CT/1600 CET. Below is the link to sign up. Hope to see everybody there!

[CFEngine: The Agent Is In](https://us02web.zoom.us/webinar/register/WN_JPz9ADvBRS2Ih8SaT0z2Uw)

**Comment by cody-valle** [comment]
ICYMI: Here is a [link](https://youtu.be/YYS1CUggomU) to the 1st session from last week where we discussed 'Debugging CFEngine Policy With Policy Analyzer'. 

We would love your feedback and/or topic ideas for future sessions.

**Comment by nickanderson** [comment]
Hey, we just did our 5th session, Running CFEngine on IoT.

![The-Agent-is-In_Episode_5_Running-CFEngine-on-IoT](https://user-images.githubusercontent.com/202896/135499444-20de55b0-0b83-4ac1-89f0-3459fccee07b.jpg)

It's not posted yet, but it when it is it will be added to the [playlist](https://www.youtube.com/watch?v=YYS1CUggomU&list=PLh71Vl9YjMaji_jnHFZ3k4mDbyOCE3A2x).

Don't forget to [subscribe](https://www.youtube.com/channel/UCAiKge7NwhuWi-KNKt3U4HA?sub_confirmation=1) if you haven't already.
